tools/run_tests/run_tests.py

```diff
@@ -440,11 +440,15 @@ def dockerfile_dir(self):
   def _get_tox_envs(self, compiler):
     """Returns name of tox environment based on selected compiler."""
     if compiler == 'default':
-      return ('py27', 'py34')
+      return ('py27', 'py34', 'py35', 'py36')
```

Comment:
@jtattermusch Currently, the dockerfiles are installing python versions using the package manager i.e. `apt-get install python-all-dev` and `apt-get install python3-all-dev`. If we want to test this code against recent python versions, we'll have to either add a [PPA] (https://launchpad.net/~fkrull/+archive/ubuntu/deadsnakes) to the package manager and that can give us at least python 3.5 (not sure if it's the dev version) or start using [pyenv](https://github.com/yyuu/pyenv) to manage the installs of python versions. Any suggestions / recommendations?